docs/src/main/asciidoc/resource-handling.adoc

```diff
@@ -85,6 +88,9 @@ NOTE: If you choose to use a security principal to authenticate and authorize wi
 
 === Basic Usage
 
+Please use this link:https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_4.0/storage[sample] as a reference for how to use Azure Spring Boot Storage Starter in your projects.
```

Comment:
After this PR is merged, it will take effect on the 4.0.0-beta.2 doc. Should we point to spring-cloud-azure_4.0.0-beta.2 instead?